app/Models/User.ts

```diff
@@ -22,6 +22,9 @@ export default class User extends BaseModel {
   @column()
   public name: string
```

Comment:
The username must to be a new column in table, the name is the google set name, so it must to be stored! but the username is a new name, how can be equals to username